### PR TITLE
Allow remote Meteor connections

### PR DESCRIPTION
**Pull request by nicolaslopezj**
This changes allows us to use remote Meteor connections to configure Apollo.

It also allows to pass a custom function to get the user stored token.

### CODE REVIEW

File: main-client.js [hunk 1]
```diff
@@ -8,9 +8,22 @@ import { _ } from 'meteor/underscore';
 const defaultNetworkInterfaceConfig = {
   path: '/graphql',
   options: {},
-  useMeteorAccounts: true
+  useMeteorAccounts: true,
+  // Pass a meteor accounts from another connection
+  meteorAccounts: Accounts,
```
Comment (lorensr) [inline main-client.js]: What's the purpose of allowing providing an Accounts from another connection? Wouldn't its _storedLoginToken still use the same localstorage key, or does it namespace it?

Comment (nicolaslopezj) [inline main-client.js]: Each Accounts object has a Meteor connection and a different login token

**Comment by lorensr** [comment]
Would you mind please linking to a demo of a remote connection working with this package in a meteor app, so I can test?

I'm also thinking `useMeteorAccounts` should now be changed to `setAuthorizationHeader`?

**Comment by helfer** [comment]
@nicolaslopezj are you still interested in this PR?

**Pull request closed without merging**